tests/tuning_tests.ts

```diff
@@ -3,16 +3,19 @@
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
 
-VF.Test.Tuning = (function () {
-  var Tuning = {
+import { Tuning } from '../src/tuning';
+import { QUnit, test, equal, expect } from './declarations';
+
+export default (function () {
+  const TuningTests = {
```

Comment:
why not just:
```typescript
return {
  ...
```

Reply:
That was my proposal on #873 but I have follow the standard you set in it with music_tests. We can still change it if you want. What we do now will be the standard. I do not mind.

Reply:
I did not mean to set any standards on music_test. It was minimal change to make it typescript compilable. Would be nice if we could get rid of this variable and just return the anonymous object.

Reply:
That was the proposal in #873, without `return TuningTests` at the end.
```
export const TuningTests = (function () {
  return {
```
If we make it simpler than that we lose the capacity to get diffs within github (git no longer recognizes that the file was renamed) and we saw that as soon as the diff was there again we identified a bug in `music_tests`

I would personally leave it make a change in the end of the migration if we want to.

Reply:
Why do you need this name anyway? export default should be just fine

Reply:
```typescript
export default (function () {
  return {
```

Reply:
`TuningTests` is used within the file to call the different functions.